solutions/src/main/java/ru/sberbank/school/task11/Streams.java

```diff
@@ -0,0 +1,73 @@
+package ru.sberbank.school.task11;
+
+import lombok.NonNull;
+
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class Streams<T> {
+
+    private final Generator<T> generatorFunction;
+
+    public Streams(Generator<T> generator) {
+        this.generatorFunction = generator;
+    }
+
+    public static <T> Streams<T> of(T... elements) {
+        return new Streams<>(generatorContext -> {
+            for (T e : elements) {
+               generatorContext.accept(e);
```

Comment:
&#39;for&#39; child has incorrect indentation level 15, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1930958209>*